.ci/images_matrix.json

```diff
@@ -1,13 +1,5 @@
 {
     "android_api": [
-        {
-            "api": 28,
-            "build_tools": "28.0.3"
-        },
-        {
-            "api": 29,
-            "build_tools": "29.0.3"
-        },
```

Comment:
Is it needed to remove API 28/29 ?

Reply:
We could have kept the support of API 28-29 but it made the tests of the CI harder to implement, because we will have to duplicate the testing apps to have one supporting JDK 17, one JDK 8, one JDK 17 with NDK, etc.

Removing the two APIs is easier to maintain. And anyway the playstore force us to use a recent targetVersion, I think it's no big deal to remove them.
And if you still need to target an old version, it is still possible to use the 1.7.0 version of the docker image